accessibilityTest/Testing.ts

```diff
@@ -150,6 +150,30 @@ export const isInit = () => {
   return get(getRoot(), SearchInterface) != null;
 };
 
+export const afterQuerySelector = <T extends Element = Element>(parentNode: HTMLElement, selector: string) => {
```

Comment:
I find the name quite vague. Would `waitUntilSelectorIsPresent()` be clearer?

Reply:
Yes, I'll change it.